Orange/widgets/data/owpreprocess.py

```diff
@@ -1144,6 +1144,9 @@ def mimeData(indexlist):
         box = gui.vBox(self.controlArea, "Output")
         gui.auto_send(box, self, "autocommit", box=False)
 
+        self.info.set_input_summary(self.info.NoInput)
+        self.info.set_output_summary(self.info.NoInput)
```

Comment:
When setting output summary, output default message (`self.info.NoOutput`) should be used.